src/components/alarmList/alarmsGridConfig.js

```diff
@@ -8,7 +8,7 @@ import { gridValueFormatters } from '../pcsGrid/pcsGridConfig';
 
 const { checkForEmpty } = gridValueFormatters;
 
-export const DEFAULT_ALARM_GRID_PAGE_SIZE = 8;
+export const DEFAULT_ALARM_GRID_PAGE_SIZE = 5;
```

Comment:
in `alarmList.js` the value is 8, here it's 5. Are they supposed to be different or the same?   Also, how did you calculate the value before (8) and now (5) ?

Reply:
It's 5. Because the rows goes beyond the alarmGrid panel if it exceeds 7.